4001-5000/LIT4274AkkonuB.xml

```diff
@@ -126,267 +126,264 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <label>Stanza 5, referring to <ref type="work" corresp="LIT3591Miracle"/>
                </label>
                <ab>
-                  h a o S እዴዑ : መትረ t ነሢአ > መጥበሕተ » ሚኅመሜ
-                  አበስ፡ ከሀነ፡ ሮሜ _________
-                  ኮመ፡ ፈወስኪዮ፡ ቅድመ፡ ሚርያም፡ እኅተ፡ ሰሎሜ
-                  ፈውሲ፡ ቊስለ * ነፍስየ፡ ወጸግውኒ፡ ሱላሜ
-                  *fol. i8r b. ጊዜ፡ ተዘ^ርኩ4፡ ስ ^ኪ፡ ቀዋሜ
+                  አመ፡ እዴሁ፡ መተረ፡ ነሢኦ፡ መጥበሕተ፡ ማኅመሜ፨
```

Comment:
a general question to all stanzas: is there a reason to not use any line element? <l n="1"></l> etc. or <lb/> between the verses?

Reply:
No, it can of course be done. I added the text as quickly as possible to work better with the manuscripts.